rasa/nlu/training_data/formats/markdown.py

```diff
@@ -151,32 +136,27 @@ def _load_files(self, line: Text) -> None:
 
     def _parse_item(self, line: Text) -> None:
         """Parses an md list item line based on the current section type."""
+        import rasa.nlu.training_data.lookup_tables_parser as lookup_tables_parser
+        import rasa.nlu.training_data.synonyms_parser as synonyms_parser
+
```

Comment:
can these be module-level imports since this is called very frequently?

Reply:
unfortunately there is something wrong with the cyclic import in the whole NLU package and I haven't managed yet to resolve it.